src/scss/components/optional/_modal.scss

```diff
@@ -70,130 +85,156 @@ $modal-borderRadius     : 4px;
             }
         }
 
-        .c-modal-title {
-            @include font-size($modal-titleFontSize);
-            padding: 0 spacing(x3);
+    .c-modal-closeBtn {
+        padding: spacing();
+        position: absolute;
+        right: spacing(x1.5);
+        top: spacing(x1.5);
+        z-index: zIndex(high);
 
-            @include media('>=mid') {
-                @include font-size(large);
-            }
+        @include media('>=mid') {
+            position: fixed;
         }
 
-            .c-modal-title--spacing {
-                padding: spacing(x3) spacing(x7);
+        &:hover {
+            cursor: pointer;
+        }
+    }
 
-                @include media('>=mid') {
-                    padding: spacing(x5) spacing(x7) spacing(x2);
-                }
+        .c-modal-closeBtn--rounded {
+            border: solid 1px $white;
+            background-color: $white;
+            border-radius: 50%;
+            opacity: 0.9;
+
+            &:hover {
```

Comment:
Do we need a `focus` here too?

Reply:
It's only there to override an `o-btn--icon:hover` hover style (which only overrides the default browser hover style). I tested this using focus and active and all is fine.